bin/watcher.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-console */
+const fs = require('fs-extra');
+const {join, basename} = require('path');
+
+function isCompiling(dataStr) {
+  return dataStr.match(
+    /File change detected. Starting incremental compilation/,
```

Comment:
Is there a better way for us to determine this?

Reply:
not that I know of /shrug